### DISCUSSION TITLE
MiningRigRentals question

### DISCUSSION THREAD

**Discussion by intellq** [RainbowMiner/RainbowMiner#2165] [Q&A] [answered]
Is it possible to configure rainbow miner to create a rig in MRR, and then more rigs join this one to add hashrate, as new workers?

This is possible and easly done when mining directly to MRR, but I don't know how to do it using multiple RBM in multiple rigs.

**Comment by RainbowMiner** [comment, marked as answer]
Hi, sorry, I totally missed this one. This is possible, but only in principle. There are some drawbacks. You could simply put all the rig's names into the description on MRR (enclosed by square brackets). BUT - and here comes the drawback - RainbowMiner cannot handle profit values on this rig. I have not implemented code for that, yet. That means, from that moment you add all rig's names, you would have to do the calculations for hashrate sum and profit on your own.
So at the moment, I wouldn't recommend doing so. Until I have implemented the code to combine multiple rigs into one MRR instance.

**Comment by intellq** [comment]
Got it, I'll wait for the update. I think it's a very usefull resource, to combine hashrate from different rigs 👍
